src/App.css

```diff
@@ -40,3 +48,342 @@
 .read-the-docs {
   color: #888;
 }
+
+/* Styles for small screens */
+@media (max-width: 480px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 1rem;
+    font-weight: 100px;
+    color: honeydew;
+    padding: 1em;
+  }
+
+  .logo {
+    height: 2.5em;
+    padding: 1.5em;
+  }
+  
+  .navList {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+  
+  .navList li {
+    margin-right: 10px; /* Add space between each nav item */
+  }
+  
+  .todoForm{
+    display: flex;
+    gap: 10px;
+    margin-bottom: 200px;
+    align-items: center;
+    
+  }
+  
+  .title{
+    margin-right:100px;
+  }
+  
+  .removeButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 20px;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 2px;
+    margin: 2px;
+    text-decoration: none;
+  }
+  
+  .addButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+}
+
+
+  /* Styles for tablets */
+@media (min-width: 481px) and (max-width: 768px){
```

Comment:
I don't see styles changing at these viewport widths. Are you sure your media queries are working here?